EduNLP/I2V/i2v.py

```diff
@@ -19,11 +19,14 @@ class I2V(object):
         the tokenizer name
     t2v: str
         the name of token2vector model
+        or the typename of token2vector model
     args:
         the parameters passed to t2v
     tokenizer_kwargs: dict
         the parameters passed to tokenizer
     pretrained_t2v: bool
+        True if use pretrained models from the remote server
```

Comment:
I do not think a tag variable is need, because you can check whether the passed argument is in pre-defined link, if not, treat it as a local path

Reply:
ok, I will change this, and I wonder if the get_pretrained_i2v should drop the tag 'from_local', which i think the tag ''from_local' make it more clear for users. 
And the usage in `get_pretrained_i2v` is a little difficult, should I split `get_pretrained_i2v` into two methods like `get_local_i2v` and `get_romote_i2v`.

Reply:
I think you can keep an argument `from`, where `from='local'` will load from files and `from='remote'` will load from remote, in addition, `from=None` by default will automatically decide where to load from.

Reply:
Why it is a lttle difficult, maybe you can give me some examples?

Reply:
I see, you need the `local_type` to specify the model type, in this case, I think `overloading` is a good option.